recipe/meta.yaml

```diff
@@ -22,12 +22,12 @@ build:
 
 requirements:
   host:
-    - python >=3.5
+    - python >=3.6
     - pip
     - nodejs >=10
   run:
-    - python >=3.5
-    - jupyterlab_server >=1.0.0,<2.0.0
+    - python >=3.6
+    - jupyterlab_server >=2.0.0rc1,<3.0.0
     - notebook >=4.3.1
```

Comment:
Are we requiring notebook, or can this be removed?

Reply:
Good point. It should be possible to update the requirements according to: https://github.com/jupyterlab/jupyterlab/blob/2bade97edadea865b294eb49c8d693d4261ca404/setup.py#L144-L152